comments/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from common.mixins.model import TimestampMixin
+from django.contrib.auth.models import User
```

Comment:
so, because the `User` model can actually be anything you want it to be, builtin, custom, whatever, and they are supposed to be interchangable, you shouldn't ever import the `User` model directly. There are two utilities to use for `User`.

Here's the django docs on it:
https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#referencing-the-user-model

You'll want to use `AUTH_USER_MODEL` when setting relationships in models, so:
```
author = models.ForeignKey(AUTH_USER_MODEL)
```
and `get_user_model()` when using the User model in other places in the code outside of field defs
```
User = get_user_model()
User.objects.delete()
```